### ISSUE TITLE
`Feature Request:` Add fusion breakpoints to the `orange` output

### ISSUE THREAD

**Issue by bounlu** [hartwigmedical/hmftools#657]
Currently the breakpoints of the fusion events are not reported in the orange output. Please add breakpoint information to the json output and to the report. https://github.com/nf-core/oncoanalyser/issues/122

**Comment by charlesshale** [comment]
The Orange report is not intended to included all pipeline output. These values are available in the Linx output files.

**Comment by bounlu** [comment]
Yes the breakpoints are available in the Linx output, but there is no common key to merge with the Orange output. Hence it is not possible to link the data to complete the reportable events.
